content/validator.ttl

```diff
@@ -0,0 +1,10 @@
+@prefix : <http://glossary.orcfax.io/#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+:validator
+    skos:prefLabel "validator"@en;
+    skos:definition '''
+    Short for Orcfax validator node.
+    A validator is a member of the #consensus mechanism for Orcfax.
```

Comment:
we're missing a consensus definition so we can remove this hash or add one and reference it?